### PR TITLE
AB#4301 -- Fixed French translations for status checker

### PR DESCRIPTION
**Pull request by AdamP-HRSDC-RHDCC**
### Description
Fixed 3 French translations for status checker

### Related Azure Boards Work Items
[AB#4301](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/4301)

### CODE REVIEW

File: frontend/public/locales/fr/status.json [hunk 1]
```diff
@@ -98,12 +98,12 @@
     "form": {
       "complete-fields": "Remplir tous les champs",
       "application-code-label": "Code de demande ou numéro de client",
-      "application-code-description": "Entrez le numéro que vous avez reçu sur la page de confirmation de la demande en ligne ou le numéro figurant dans le coin supérieur droit de la lettre que vous avez peut-être reçue de Service Canada.",
+      "application-code-description": "Entrez le numéro que vous avez reçu aprés avoir demande en ligne ou le numéro figurant dans le coin supérieur droit de la lettre que vous avez peut-être reçue de Service Canada.",
```
Comment (sebastien-comeau) [inline frontend/public/locales/fr/status.json:101]:
```suggestion
      "application-code-description": "Entrez le numéro que vous avez reçu après avoir demande en ligne ou le numéro figurant dans le coin supérieur droit de la lettre que vous avez peut-être reçue de Service Canada.",
```

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]

File: frontend/public/locales/fr/gcweb.json [hunk 2]
```diff
@@ -112,7 +112,7 @@
   },
   "error-summary": {
     "header_one": "L'erreur suivante a été trouvée dans le formulaire\u00a0:",
-    "header_other": "Les {{count}} erreurs suivantes ont été trouvées dans le formulaire\u00a0:"
+    "header_other": "Les {{count}} erreurs suivantes ont été trouvées dans le formulaire:"
```
Comment (nicholas-ly) [inline frontend/public/locales/fr/gcweb.json:115]: The non-breaking space should be left in before the colon in my opinion. The ADO doesn't seem to reference a Figma - I think we should make the argument to leave this as is.

Comment (sebastien-comeau) [reply frontend/public/locales/fr/gcweb.json:115]: Good call. French always have a space before colon.

File: frontend/public/locales/fr/status.json (hunk 1 above)
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/fr/status.json]: Fixed typo

File: frontend/public/locales/fr/gcweb.json (hunk 2 above)
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/fr/gcweb.json]: Re-added space

**Review by nicholas-ly — APPROVED** [review]

**Pull request merged**